overview/default.nix

```diff
@@ -301,9 +300,7 @@ let
             setTimeout(() => button.textContent = "Copy", 2000);
           }
 
-          ${
-            "" # TODO: this should be the exact same code for copying file content
-          }
+          // TODO: this should be the exact same code for copying file content
```

Comment:
Why? This comment should not show up in the file output, and changing it will require a rebuild for no good reason.